cartridge/vshard-utils.lua

```diff
@@ -266,6 +266,8 @@ local function validate_config(conf_new, conf_old)
                 validate_group_upgrade(name, topology_new, topology_old)
             end
         end
+
+        topology.validate_distances(conf_new.zone_distances);
```

Comment:
I suggest moving it's implementation to this module (similar to `validate_group_weights`)